### ISSUE TITLE
Do you have a plane to Change the name of API ?

### ISSUE THREAD

**Issue by finscn** [flyover/box2d.ts#7]
@flyover , Hi ,
Do you have a plane to Change the name of API , then provide a JavaScript-friendly API ?

example:
box2d.b2BodyDef --> Box2d.bodyDef
world.CreateBody --> world.createBody
b2BodyType.b2_dynamicBody --> bodyType.dynamicBody
...

**Comment by Nick-Riggs** [comment]
This lib is looking awesome and it's performance is very good. 

👍 I would also like to see the naming conventions more along the lines of ECMAScript standards.

**Comment by Nek-** [comment]
On my side, I'm in favor of using Box2D standards. To improve code portability.

**Comment by alpertuna** [comment]
I also don't like the Box2D API naming BUT to benefit Box2D manual and tuts on internet, changing API may not be a good idea. Instead I have some wrapper classes on my project to use Box2D easily.
Btw thank you! for preparing this ts version, so that I can find my way using autocompletions.

**Comment by flyover** [comment]
Yeah, I think I'm going to leave it as close to the original API as possible.  Perhaps a wrapper that added some of these JS features would be nice, especially if the underlying physics engine could be swapped out.
